### PR TITLE
Issue / 8563 Extends Ads Settings View PAX

### PR DESCRIPTION
**Pull request by 10upsimon**
## Summary

Addresses issue:

- #8563

## Relevant technical choices

- Updated logic so that PAX view settings show if only one PAX setting has a value
- Added logic for gating PAX ads settings view behind feature flag
- Updated PAX ads settings story to use feature flag

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 5.2 and PHP 7.4.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [x] Run the code.
- [x] Ensure the acceptance criteria are satisfied.
- [x] Reassess the implementation with the IB.
- [x] Ensure no unrelated changes are included.
- [x] Ensure CI checks pass.
- [x] Check Storybook where applicable.
- [x] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/js/modules/ads/components/settings/SettingsView.js [hunk 1]
```diff
@@ -52,50 +55,62 @@ export default function SettingsView() {
...
-							<DisplaySetting value={ extCustomerID } />
-						) : (
-							__( 'None', 'google-site-kit' )
-						) }
-					</p>
-				</div>
-			) }
+			{ ! isAdBlockerActive &&
+				paxEnabled &&
+				( paxConversionID || extCustomerID ) && (
```
Comment (zutigrm) [inline assets/js/modules/ads/components/settings/SettingsView.js:100]: This conditional check is pretty crowded, making it a bit harder to follow. Perhaps we can make a new variable `shouldRenderExtCustomerID = paxEnabled && ( paxConversionID || extCustomerID )` and then use that single variable as only check at this point to determine if markup should be rendered.

**Review by zutigrm — CHANGES_REQUESTED** [review]
Thanks @10upsimon , nice work. I left you a small feedback

File: assets/js/modules/ads/components/settings/SettingsView.js [hunk 2]
```diff
@@ -52,50 +55,62 @@ export default function SettingsView() {
...
-							<DisplaySetting value={ paxConversionID } />
-						) : (
-							__( 'None', 'google-site-kit' )
-						) }
-					</p>
-				</div>
-			) }
+			{ ! isAdBlockerActive &&
+				paxEnabled &&
+				( paxConversionID || extCustomerID ) && (
```
[the remark this answers is not in the document]
Comment (10upsimon) [reply assets/js/modules/ads/components/settings/SettingsView.js:80]: @zutigrm this was amended as part of QA feedback from @mohitwp . Previously it functioned the same as how you are asking if it should.

File: assets/js/modules/ads/components/settings/SettingsView.js [hunk 3]
```diff
@@ -52,50 +55,62 @@ export default function SettingsView() {
...
-							<DisplaySetting value={ conversionID } />
-						) : (
-							__( 'None', 'google-site-kit' )
-						) }
-					</p>
-				</div>
-			) }
+			{ ! isAdBlockerActive &&
+				( ( ! paxConversionID && ! extCustomerID ) ||
+					! paxEnabled ) && (
```
[the remark this answers is not in the document]
Comment (10upsimon) [reply assets/js/modules/ads/components/settings/SettingsView.js]: Good spot @zutigrm I have greatly simplified this.

File: assets/js/modules/ads/components/settings/SettingsView.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (10upsimon) [reply assets/js/modules/ads/components/settings/SettingsView.js]: Let's leave this as is as per the internal discussion, we can re-visit this during the initial phase if need be.

File: assets/js/modules/ads/components/settings/SettingsView.js (hunk 1 above)
Comment (10upsimon) [reply assets/js/modules/ads/components/settings/SettingsView.js]: I've simplified this somewhat @zutigrm

**Review by zutigrm — APPROVED** [review]
Thanks @10upsimon LGTM ✅

**Review by tofumatt — APPROVED** [review]

**Pull request merged**